.github/actions/lint/action.yml

```
@@ -0,0 +1,28 @@
+name: golangci-lint-k6
+description: Run golangci-lint with the k6 ruleset
+
+runs:
+  using: composite
+  steps:
+    - name: Install Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: 1.20.x
+        check-latest: true
+    - name: Get golangci-lint version and download rules
+      shell: bash
+      env: #TODO: remove after https://github.com/actions/runner/issues/2473 is fixed
+        GITHUB_ACTION_REF: ${{ github.action_ref || github.head_ref }}
+      run: |
+        RulesURL="https://raw.githubusercontent.com/grafana/k6/${{ env.GITHUB_ACTION_REF }}/.golangci.yml"
+        echo "Downloading ${RulesURL}..."
+        curl "${RulesURL}" --output "${{ github.action_path }}/.golangci.yml"
```

```suggestion
        curl -fsSL "${RulesURL}" --output "${{ github.action_path }}/.golangci.yml"
```

I instinctively use these flags in scripts. It's silent and safer.

---

Resolved by the [latest force-push](https://github.com/grafana/k6/compare/09b47c9c6e9dda7c323ed20a5389d4860df2025e..6de4f1c4c2666717f116b047759f4ee47bee0d9a)?

Though I used the long-form options, and I changed `-L` to its opposite, `--no-location`, since I don't want curl to follow redirects in this case.